### ISSUE TITLE
support simple vector drawing

### ISSUE THREAD

**Issue by robaho** [robaho/seashore#174]
tracking issue to support basic vector drawing (e.g. arrows, boxes, ellipsis, etc.)

Often when marking up (annotating) a photo image being able to draw/move using vector shapes is easier.

**Comment by theses-sprint** [comment]
When picking a workspace for images that I will need to use in that way, I wouldn't choose Seashore anyway; there are other lightweight tools or serious tools that already fill that use case.

**Comment by cwjordan** [comment]
I agree with theses-sprint, while adding basic vector drawing is a cool feature, I don't think I'd actually use it myself.  When I annotate something I always just do it quickly in Preview.  Other people will have very different use cases of course - I only use Seashore for small things like quickly cropping or scaling an image, or very minor editing.

**Comment by MagnetarRocket** [comment]
I would like that very much as currently vectornator has the gremlins on my currently used mac.

**Comment by victor-marino** [comment]
Just chiming in to say I do support this feature request.

It's really hard to find a free, lightweight and easy to use app in Mac for these basic tasks, and I'd love to see Seashore supporting it so I could get rid of more complex apps.

It's true that the "Preview" app does support some basic vector drawing, but sometimes you just want to create a new picture and draw a couple of squares on it, maybe add a second layer for text, etc.

I do agree with the general spirit that the priority should be to keep Seashore simple in that sense, so nothing too advanced. But a simple drawing tool where we can create lines, arrows, rectangles, circles, etc. would be very useful in my opinion.

**Comment by theses-sprint** [comment]
I have never had any problems on Mac with the simplicity of Wobastic's vector drawing tool Omber for the uses you describe (although I see that 4 people on iOS app store say that the mobile version has stopped working this year), if that helps.[1]
I wouldn't want Seashore to become one of those more broad and complex apps that would be great to get rid of.

[1] https://www.wobastic.com/omber/

**Comment by robaho** [comment]
Closing as this is available in Seahorse, and Seashore is deprioritized.
